go/cmd/dolt/commands/checkout.go

```diff
@@ -266,20 +316,21 @@ func checkoutBranch(ctx context.Context, dEnv *env.DoltEnv, name string, force b
 
 // setRemoteUpstreamForCheckout sets upstream for checked out branch. This applies `dolt checkout <bn>`,
 // if <bn> matches any remote branch name. This should not happen for `dolt checkout -b <bn>` case.
-func setRemoteUpstreamForCheckout(dEnv *env.DoltEnv, branchRef ref.RemoteRef) errhand.VerboseError {
-	refSpec, err := ref.ParseRefSpecForRemote(branchRef.GetRemote(), branchRef.GetBranch())
+func setRemoteUpstreamForCheckout(dEnv *env.DoltEnv, remName, remBranchName string) errhand.VerboseError {
```

Comment:
(minor/nit) `remote` and `remoteBranch` might be easier to read variable names. I stopped for a second to think about what `remName` meant. It's not a bad name, but calling it `remote` would be even more obvious I think.